blkn/src/Button/IconWrapper.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  size: "normal" | "small" | "large",
+  type: "primary" | "secondary",
```

Comment:
I would use the `$Keys<T>` type for this (see [Flow utility types](https://flow.org/en/docs/types/utilities/)):

```
size: $Keys<typeof sizes>,
type: $Keys<typeof fillColors>,
```